boards/st/stm32f411e_disco/stm32f411e_disco_stm32f411xe_B.overlay

```diff
@@ -8,6 +8,7 @@
 	aliases {
 		/delete-property/ magn0;
```

Comment:
I wanted to ask why the magnetometer was removed here even when a new one is defined below.  I don't have a B-revision board to test on, so I won't change it.  Just curious.